cluster/directory/static_directory.go

```diff
@@ -53,8 +58,16 @@ func (dir *staticDirectory) IsAvailable() bool {
 }
 
 func (dir *staticDirectory) List(invocation protocol.Invocation) []protocol.Invoker {
-	//TODO:Here should add router
-	return dir.invokers
+	invokers := dir.invokers
```

Comment:
either u use the lock to protect dir.invokers or u just copy it in another func.